src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/duration.pipe.ts

```diff
@@ -1,18 +1,12 @@
 import { Pipe, PipeTransform } from '@angular/core';
 
-import moment from 'moment';
-
 @Pipe({
   name: 'duration',
   pure: false
 })
 export class DurationPipe implements PipeTransform {
-  transform(date: any, isRelative = false): string {
-    if (isRelative) {
-      return moment(date).fromNow();
-    } else {
-      return this._forHumans(date);
-    }
+  transform(seconds: number): string {
+    return this._forHumans(seconds);
```

Comment:
Just an observation: if the intention of the underscore in ```_forHumans``` method is to mark it as not public, what about making it private: ```private static forHumans(...``` ?